Calculator/Program.cs

```diff
@@ -1,12 +1,21 @@
 ﻿using System;
 
-namespace Calculator
+namespace CalculatorOOPS
 {
-    class Program
+    public static class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            try
+            {
+                string a = Keystore.MenuFilePath;
+                ReadAndWriteInConsole.ReadMenuforCalcType(a);
```

Comment:
ReadMenuforCalcType method will only read menu and do nothing else.
So why do we need to pass this parameter here?
Cant you use Keystore.MenuFilePath inside the ReadMenuforCalcType itself?